### DISCUSSION TITLE
License of ID's ambulance station icon

### DISCUSSION THREAD

**Discussion by wolfgang-8** [openstreetmap/iD#10584] [Q&A] [answered]
I hope this is the right place to ask this question.

[Here](https://github.com/gravitystorm/openstreetmap-carto/issues/3968#issuecomment-985675818), the idea came up to also use ID's ambulance station icon 
![image](https://github.com/user-attachments/assets/c226f721-257c-4caf-8373-04cc8bea38a1)
in osm-carto. What is the source and license of this icon and is it possible to use it in carto?

Thanks a lot!

**Comment by 1ec5** [comment, marked as answer]
Preset icons are defined in the id-tagging-schema repository along with other preset metadata:

https://github.com/openstreetmap/id-tagging-schema/blob/c1ece646b2e049d20c0d3658e521a65649fdd084/data/presets/emergency/ambulance_station.json#L2

Icons can come from [several sources](https://github.com/ideditor/schema-builder/blob/main/ICONS.md#icons). Based on the `fas-` prefix in that JSON file, it comes from Font Awesome’s free tier, which is licensed under [Creative Commons Attribution-ShareAlike 4.0](https://github.com/FortAwesome/Font-Awesome?tab=readme-ov-file#license). According to the documentation, attribution is already included if you use the original SVG file.

**Reply by wolfgang-8** [reply]
Thanks for the detailed information!
